src/scenes/Activities/ActivityHistory/styles.js

```diff
@@ -1,6 +1,7 @@
 const styles = () => ({
-  historyTable: {
-    overflowX: 'scroll',
+  historyTableWrapper: {
+    width: '100%',
+    overflowX: 'auto',
```

Comment:
can these be applied to all tables?

Reply:
Seems that it needs to apply to the table's parent element, not table element itself.

Reply:
gotcha. ok.